x-pack/plugins/license_management/__jest__/__snapshots__/license_status.test.js.snap

```diff
@@ -1,5 +1,5 @@
 
-exports[`LicenseStatus component should display normally when license is active 1`] = `"<div><div class=\\"euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--justifyContentSpaceAround euiFlexGroup--directionRow euiFlexGroup--responsive\\"><div class=\\"euiFlexItem euiFlexItem--flexGrowZero\\"><div class=\\"euiFlexGroup euiFlexGroup--gutterSmall euiFlexGroup--alignItemsCenter euiFlexGroup--directionRow euiFlexGroup--responsive\\"><div class=\\"euiFlexItem euiFlexItem--flexGrowZero\\"><svg class=\\"euiIcon euiIcon--medium euiIcon--success\\" width=\\"16\\" height=\\"16\\" viewBox=\\"0 0 16 16\\" xmlns=\\"http://www.w3.org/2000/svg\\"><path d=\\"M8 16A8 8 0 1 1 8 0a8 8 0 0 1 0 16zm-1.125-5a.377.377 0 0 0 .266-.11l5.25-5.25a.376.376 0 0 0-.531-.53l-4.985 4.985L4.141 7.36a.376.376 0 0 0-.531.53l3 3c.073.073.17.11.265.11z\\" fill-rule=\\"evenodd\\"></path></svg></div><div class=\\"euiFlexItem euiFlexItem--flexGrowZero\\"><h2 class=\\"euiTitle euiTitle--large\\">Your Gold license is active</h2></div></div></div></div><div class=\\"euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--justifyContentSpaceAround euiFlexGroup--directionRow euiFlexGroup--responsive\\"><div class=\\"euiFlexItem euiFlexItem--flexGrowZero\\"><div class=\\"euiText\\"><span class=\\"euiTextColor euiTextColor--subdued\\"><span>Your license will expire on <strong>October 12, 2099 7:00 PM EST</strong></span></span></div></div></div><div class=\\"euiSpacer euiSpacer--l\\"></div></div>"`;
+exports[`LicenseStatus component should display normally when license is active 1`] = `"<div><div class=\\"euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--justifyContentSpaceAround euiFlexGroup--directionRow euiFlexGroup--responsive\\"><div class=\\"euiFlexItem euiFlexItem--flexGrowZero\\"><div class=\\"euiFlexGroup euiFlexGroup--gutterSmall euiFlexGroup--alignItemsCenter euiFlexGroup--directionRow euiFlexGroup--responsive\\"><div class=\\"euiFlexItem euiFlexItem--flexGrowZero\\"><svg class=\\"euiIcon euiIcon--medium euiIcon--success\\" focusable=\\"false\\" width=\\"16\\" height=\\"16\\" viewBox=\\"0 0 16 16\\" xmlns=\\"http://www.w3.org/2000/svg\\"><path d=\\"M8 16A8 8 0 1 1 8 0a8 8 0 0 1 0 16zm-1.125-5a.377.377 0 0 0 .266-.11l5.25-5.25a.376.376 0 0 0-.531-.53l-4.985 4.985L4.141 7.36a.376.376 0 0 0-.531.53l3 3c.073.073.17.11.265.11z\\" fill-rule=\\"evenodd\\"></path></svg></div><div class=\\"euiFlexItem euiFlexItem--flexGrowZero\\"><h2 class=\\"euiTitle euiTitle--large\\">Your Gold license is active</h2></div></div></div></div><div class=\\"euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--justifyContentSpaceAround euiFlexGroup--directionRow euiFlexGroup--responsive\\"><div class=\\"euiFlexItem euiFlexItem--flexGrowZero\\"><div class=\\"euiText\\"><span class=\\"euiTextColor euiTextColor--subdued\\"><span>Your license will expire on <strong>October 12, 2099 7:00 PM EST</strong></span></span></div></div></div><div class=\\"euiSpacer euiSpacer--l\\"></div></div>"`;
```

Comment:
Maybe not something to do in this PR, but could we figure out how to add some line breaks to this snapshot? It's hard to compare diffs when it's all on one line.